server/parsec/components/postgresql/organization_test_dump_organizations.py

```diff
@@ -22,7 +23,9 @@
     is_expired,
     active_users_limit,
     user_profile_outsider_allowed,
-    minimum_archiving_period
+    minimum_archiving_period,
+    tos_updated_on,
+    tos_url
```

Comment:
`tos_per_locale_urls`?
```suggestion
    tos_per_locale_urls
```

Reply:
done.